src/renderers/shared/stack/event/EventPluginRegistry.js

```diff
@@ -7,35 +7,46 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule EventPluginRegistry
+ * @flow
  */
 
 'use strict';
 
+type PluginName = string;
+
+type PluginModule = {
+  eventTypes: {},
+};
+
+type NamesToPlugins = {[key: PluginName]: PluginModule};
+
+type EventPluginOrder = ?Array<PluginName>;
```

Comment:
I think we're wanting to use ` | null` instead of `?` so we avoid the ambiguity that the maybe type has https://www.facebook.com/groups/2003630259862046/permalink/2098952973663107/